source/administration/changelog.md

```diff
@@ -2,6 +2,224 @@
+- Updated the channel info dialog to be more user friendly.
+
+**LDAP (Enterprise)**
+
+- Added a button to manually trigger LDAP synchronization.
+- Updating Synchronization Interval no longer requires a server restart to take effect.
+- Improved logging for LDAP synchronization.
+- Added validation to the LDAP settings in the System Console so an error is triggered if required fields are missing.
+
+**Enterprise**
+
+- [New command line tools](http://docs.mattermost.com/administration/command-line-tools.html) added, such as inviting and removing users from channels, and restoring previously deleted channels.
+- Added the ability to set different themes per team.
+
+### Bug Fixes
+
+- Smileys ending in a letter are now prioritized first in the autocomplete list.
+- Clarified the GitLab SSO error message if another Mattermost account is already associated with the GitLab account.
+- Privacy settings in the system console now refresh correctly when hiding email addresses or full names.
+- Fixed the cross contamination of new channels created on different teams in the same browser.
+- Create channel modal now saves upon pressing `ENTER` or `CTRL+ENTER` depending on the user setting.
+- Team creation via GitLab SSO no longer throws an error if email domains are restricted.
+- Channel header no longer disappears after renaming a channel
+- Testing the email connection in the System Console no longer throws an error.
+- Multiline list items are now displayed correctly on new lines.
+- Error message is updated when switching from email to GitLab SSO authentication that is already used by another account.
+- Timestamps no longer require a page refresh when switching between 12h and 24h display formats.
+- `#in`, `#from`, or `#channel` now highlight in search results.
+- No longer require a page refresh before enabling compliance reporting in the System Console.
+- `@all` no longer sends mentions if unselected in Account Settings.
+- Users are no longer redirected to the switch teams page after changing authentication method from GitLab SSO to email.
+- Invalid MFA token error message now clears correcly from the UI.
+- Errors now correcly clear from the UI when changing passwords.
+- System Console users list no longer throws an error when trying to demote a member from a System Admin.
+- iOS radio buttons no longer stay selected when switching between options.
+- Email addresses now display for System Admins even if hidden in the System Console.
+- Code themes now save when updated via Account settings.
+- file name is now displayed inctead of the full path to the file in code snippet previews.
+- Config settings are refreshed immediately when **Reload Configuration from Disk** is clicked.
+- Preview feature checkboxes now reset after changes are canceled.
+- Updated the markdown parser to fix poor handling of certain links.
+- Right-hand sidebar no longer appears black in some cases on IE11 and Edge.
```

Comment:
Was this in v3.1? Can we do a double check of the bugs list and remove anything that wasn't actually a bug in v3.1, and was only introduced afterwards?